### PR TITLE
SV-COMP 2023 setup

### PR DESCRIPTION
**Pull request by sim642**
This PR conflicts with #845 due to the changes in stubs structure.

### TODO
- [x] Add autotune conf

### CODE REVIEW

File: conf/svcomp23.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      "activated": [
+        "singleThreaded",
+        "mallocWrappers",
+        "noRecursiveIntervals",
+        "enums",
+        "arrayDomain",
+        "octagon",
+        "wideningThresholds"
+      ]
+    }
```
Comment (sim642) [inline conf/svcomp23.json:70]: @leunam99 Is this the set of autotune activations that you'd recommend?

Comment (leunam99) [reply conf/svcomp23.json:70]: Yes it is. One caveat is that this configuration also slightly increased the number of test with a wrong result in my benchmarks. 
(` sv-benchmarks/c/systemc/pipeline.cil-1.yml,`
`sv-benchmarks/c/ldv-linux-3.14-races/linux-3.14--drivers--usb--misc--iowarrior.ko.cil.yml`,
`sv-benchmarks/c/ldv-validator-v0.6/linux-stable-af3071a-1-130_7a-drivers--hwmon--s3c-hwmon.ko-entry_point.cil.out.yml`
and another one I would have to look up later today)

Comment (sim642) [reply conf/svcomp23.json:70]: - [x] `systemc/pipeline.cil-1` is fixed by https://github.com/goblint/analyzer/pull/859/commits/2c257b60cde8c6b9214f378d92ee82c4f66ba458.
- [x] `ldv-linux-3.14-races/linux-3.14--drivers--usb--misc--iowarrior.ko.cil` is already wrong on master due to a task problem: https://gitlab.com/sosy-lab/benchmarking/sv-benchmarks/-/merge_requests/1374.
- [ ] `ldv-validator-v0.6/linux-stable-af3071a-1-130_7a-drivers--hwmon--s3c-hwmon.ko-entry_point.cil.out` is part of #780.

**Comment by michael-schwarz** [comment]
We should also think about whether to enable the analysis that can keep more malloc'ed variables apart #722.

**Comment by sim642** [comment]
> We should also think about whether to enable the analysis that can keep more malloc'ed variables apart #722.

According to https://github.com/goblint/analyzer/pull/722#issuecomment-1196424419, it made no difference, with 3 at least. If we want some higher value, we should do local sv-benchmarks experiments first (because the preruns turnaround time is much longer and they're currently limited to 60s anyway) to find such optimal value. Or maybe autotune needs to be adapted to also tune that.

Since this setup has successfully passed a prerun, the Apron packaging stuff works, which was the main concern. I'll merge this so conflicts in #845 can be resolved.

**Pull request merged**